R/elasticsearch_eda_funs.R

```diff
@@ -44,6 +45,24 @@ get_counts <- function(field
     # Input checking
     es_host <- .ValidateAndFormatHost(es_host)
     
+    # Other input checks we don't have explicit error messages for
+    .assert(
+        assertthat::is.string(field)
+        , field != ""
+        , assertthat::is.string(es_index)
+        , es_index != ""
+        , assertthat::is.string(start_date)
+        , start_date != ""
+        , assertthat::is.string(end_date)
+        , end_date != ""
+        , assertthat::is.string(time_field)
+        , time_field != ""
+        , assertthat::is.string(use_na)
+        , use_na != ""
+        , assertthat::is.count(max_terms)
+        , max_terms > 0
```

Comment:
This is redundant - `is.count` ensures it's 1 or greater

Reply:
whoops, yep will fix